packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -702,6 +699,20 @@ const NavBar = <AdditionalEnvironmentProperties extends {}>(
           })}
         </div>
         <div className={styles['fixed-menu']}>
+          <div
+            css={css`
+              position: absolute;
+              top: -32px;
+              height: 32px;
+              width: 100%;
+              background: linear-gradient(
+                0deg,
+                ${appkitDesignTokens.backgroundColorForNavbar} 0%,
+                rgba(0, 0, 0, 0) 100%
+              );
+              z-index: 1;
+            `}
+          />
```

Comment:
**nit**: perhaps we could have a local styled component for this. Perhaps something like `MenuItemContainer` or `MenuItemWraper`

Reply:
See https://github.com/commercetools/merchant-center-application-kit/pull/2897/commits/996fb1f9c79809c5dd585bf4345c0032d2b47a8b